### DISCUSSION TITLE
Legibility

### DISCUSSION THREAD

**Discussion by esginmurat** [ubuntu/yaru#3979] [Q&A] [answered]
Legibility

Hi. First of all, thank you to the developers for this beautiful theme.

In black theme on Ubuntu 22.04 and gnome 42, nautilus background colors are as in image image: 
![nautilus-dark](https://github.com/ubuntu/yaru/assets/17769013/ddb50f86-93af-4fc2-abd5-cbb1ed0e17fe)

But on Ubuntu 23x and gnome 44 the colors are like here: 

![Screenshot from 2023-09-03 13-51-58](https://github.com/ubuntu/yaru/assets/17769013/1ec918bc-e9bb-4335-a5c1-efa0f3c76102)

The colors of the left side and the middle part have changed places. However, in terms of readability, the one like the 22.04 gnome picture is more readable because the dark color is slightly opened and does not tire the eyes.

1. Can we use the "yaru dark theme" colors of the old gnome 42 ubuntu 22.04 in gnome 44 ubuntu 23x?
Secondly, can we make the white a little more faded so that the white text colors in the dark theme are a little less eye strain? Not completely white, but a bit more like a grayish version of white?

Thanks.

**Comment by Jupi007** [comment, marked as answer]
Hello @esginmurat and thank you for your message.
I'm really sorry, but we don't have any control on the new nautilus theming.
On your first screenshot, you show the Gtk3 version of nautilus, which was themable by us.
But the new nautilus version shipped in Ubuntu 23.10 uses Gtk4 + Libhandy, which lock all theming features.

So I need to close this, as we do not have any control on this.

**Comment by esginmurat** [comment]
First of all, thank you for the response. I understood why. This issue I mentioned will be a problem in future LTS versions. Thank you again. enjoy your work.

**Comment by Feichtmeier** [comment]
Eventually we could tweak the colors to have a more small difference

**Reply by Jupi007** [reply]
Do you mean to tweak the Gtk3 theme colors?
